_i18n/en/_posts/projects/2017-10-07-projekt-tensor-hive.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: article-post
+locale: en
+title:  "TensorHive"
+date:   2017-10-07 12:00:00
+author: Paweł Rościszewski
+categories: Projects
+tags:	project tesnsorhive
+comments: true
+cover:  "/assets/projects/tensorhive/cover.jpg"
+thumbnail: "/assets/projects/tensorhive/thumbnail.jpg"
+---
+
+W przypadku dużych architektur neuronowych i zbiorów danych treningowych, treningi mogą trwać wiele godzin. Na przykład w projekcie AlphaGo, sieć do klasyfikacji pozycji w grze Go na podstawie archiwalnych ruchów ekspertów była trenowana przez trzy tygodnie na 50 GPU. Rozproszenie treningu na wiele urządzeń jest nieuniknione w wielu praktycznych zastosowaniach. Choć trening sieci neuronowych jest problemem trudnym do zrównoleglenia, popularne frameworki do uczenia głębokiego udostępniają pewne mechanizmy treningu rozproszonego.
```

Comment:
Ma być po polsku? Wygląda na brak tłumaczenia. Jeśli nie mamy go to może chociaż zasygnalizujmy to jakimś `[Polish only]`?

Reply:
Hmmm.. Faktycznie. Pominąłem coś...

Reply:
Okej. Zabrakło jednego tłumaczenia, powinno być do końca jutrzejszego dnia 😉